### PR TITLE
Landingpage   404 page

### PR DESCRIPTION
**Pull request by ark2002**
- Landing page added
- 404 page added

### CODE REVIEW

File: src/frontend/screens/LandingScreen/LandingScreen.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
+.landing-page__container {
+    align-items: center;
+}
+
+.landing-page__hero {
+    height: 19%;
+    width: 19%;
+    min-width: 300px;
```
Comment (chanchal-panpaliya) [inline src/frontend/screens/LandingScreen/LandingScreen.css:8]: use rem

File: src/frontend/screens/PageNotFoundScreen/PageNotFoundScreen.js [hunk 2]
```diff
@@ -1,9 +1,16 @@
...
 const PageNotFoundScreen = () => {
 
     return (
-        <>
-            PageNotFoundScreen
-        </>
+        <div className="error-page__container flex--column">
+            <div className="page__title flex--row" >
+                <h1 className="page__main primary__font">4</h1>
+                <img src="/assets/zodiaclogo.png" alt="logo" className="page__hero" />
```
Comment (chanchal-panpaliya) [inline src/frontend/screens/PageNotFoundScreen/PageNotFoundScreen.js:9]: alternative way , import img_404 from "../../imgpath" ; then 404_img

**Review by chanchal-panpaliya — APPROVED** [review]
code looks clean
proper naming
good UI
good folder structure

**Review by ranitasahaa1230 — APPROVED** [review]
Code looks clean
Looks good to merge

File: src/frontend/screens/LandingScreen/LandingScreen.css (hunk 1 above)
Comment (ark2002) [reply src/frontend/screens/LandingScreen/LandingScreen.css:8]: done

**Pull request merged**